components/plugin/src/miniboxing/plugin/metadata/InteropDefinitions.scala

```diff
@@ -156,4 +156,25 @@ trait InteropDefinitions {
 
   lazy val nobridgeClass = global.rootMirror.getRequiredClass("miniboxing.runtime.nobridge")
   lazy val nobridgeTpe = nobridgeClass.tpe
+
+  // Type classes
+
+  lazy val MiniboxedNumericClass = global.rootMirror.getRequiredClass("miniboxing.runtime.math.MiniboxedNumeric")
+  lazy val MiniboxedOrderingClass = global.rootMirror.getRequiredClass("miniboxing.runtime.math.MiniboxedOrdering")
+  lazy val MiniboxedOrderedClass = global.rootMirror.getRequiredClass("miniboxing.runtime.math.MiniboxedOrdered")
+  lazy val MiniboxedIntegralClass = global.rootMirror.getRequiredClass("miniboxing.runtime.math.MiniboxedIntegral")
+  lazy val MiniboxedFractionalClass = global.rootMirror.getRequiredClass("miniboxing.runtime.math.MiniboxedFractional")
+  lazy val NumericClass = global.rootMirror.getRequiredClass("scala.math.Numeric")
+  lazy val OrderingClass = global.rootMirror.getRequiredClass("scala.math.Ordering")
+  lazy val OrderedClass = global.rootMirror.getRequiredClass("scala.math.Ordered")
+  lazy val IntegralClass = global.rootMirror.getRequiredClass("scala.math.Integral")
+  lazy val FractionalClass = global.rootMirror.getRequiredClass("scala.math.Fractional")
+  lazy val TypeClasses: Map[Symbol, Symbol] =
+    Map (
+      NumericClass    -> MiniboxedNumericClass,
+      OrderingClass   -> MiniboxedOrderingClass,
+      OrderedClass    -> MiniboxedOrderedClass,
+      IntegralClass   -> MiniboxedIntegralClass,
+      FractionalClass -> MiniboxedFractionalClass
+    )
```

Comment:
To avoid the duplication, you can add this code to [`CommonDefinitions.scala`](https://github.com/miniboxing/miniboxing-plugin/blob/wip/components/plugin/src/miniboxing/plugin/metadata/CommonDefinitions.scala)